components/header-nav.tsx

```diff
@@ -0,0 +1,114 @@
+import { Disclosure } from "@headlessui/react";
+import { MenuIcon, XIcon } from "@heroicons/react/outline";
+import Discord from "@components/svg/discord-svg";
+import TwitterIcon from "@components/svg/twitter-icon-svg";
+import Image from "next/image";
+
+const navigation = [
+  { name: "Home", href: "https://www.infinitykeys.io/future/landing" },
```

Comment:
```suggestion
  { name: "Home", href: "/" },
```

Please update the the home link to lead to https://infinitykeys.io

We can achieve this by hitting the default route "/".

We cannot hardcode `/future/landing` route as this will not be correct once the marketing page will be live at infinitykeys.io/